content/en/serverless/_index.md

```diff
@@ -34,17 +34,54 @@ further_reading:
 
 {{< vimeo 543362476 >}}
 
+ 
+<div class="alert alert-info">Make sure to check out discussions going on in the <a href="https://datadoghq.slack.com/archives/CFDPB83M4">#serverless</a> channel in the <a href="https://chat.datadoghq.com/">Datadog Slack community</a>.</div>
 
-Serverless is a concept where you write event-driven code and upload it to a cloud provider, which manages all of the underlying computational resources. [Datadog Serverless][1] brings together metrics, traces, and logs from your AWS Lambda functions running serverless applications into one view.
+[Datadog Serverless Monitoring][1] provides full visibility into all of the managed services that power your serverless applications by bringing together real-time metrics, logs, and traces from your Lambda functions as well as related fully-managed APIs, queues, streams, and data stores.
 
-<div class="alert alert-info">Make sure to check out discussions going on in the <a href="https://datadoghq.slack.com/archives/CFDPB83M4">#serverless</a> channel in the <a href="https://chat.datadoghq.com/">Datadog Slack community</a>.</div>
+## Quick start
+
+If you are new to Datadog, you can get started by [signing up for a Datadog account][5], then following the instructions for [installing Datadog's Lambda Extension][6]. These steps configure your Lambda functions to send real-time metrics, logs, and traces to Datadog:
+
+{{< img src="serverless/aws-fully-managed-services-serverless-monitoring-hero.png" alt="Datadog Serverless Monitoring"  style="width:100%;" >}}
+
+## Installation instructions
 
-## Getting started
+For more detailed installation instructions, select the Lambda runtime below for instructions to instrument your serverless application:
 
-Select the Lambda runtime below for instructions to instrument your serverless application.
 {{< partial name="serverless/getting-started-languages.html" >}}
 
-## Other services
+## Explore Datadog Serverless Monitoring
```

Comment:
Does it make sense to include the links to the corresponding blog posts here in case user has a strong interest in one of the use cases?